x-pack/solutions/search/plugins/search_homepage/public/components/dive_deeper/dive_deeper_with_elasticsearch.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+  useCurrentEuiBreakpoint,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { docLinks } from '../../../common/doc_links';
+import { DocCallouts } from './doc_callouts';
+
+export const DiveDeeperWithElasticsearch: React.FC = () => {
+  const currentBreakpoint = useCurrentEuiBreakpoint();
+  return (
+    <>
+      <EuiTitle size="xxs">
+        <h4>
+          {i18n.translate('xpack.searchHomepage.diveDeeper.title', {
+            defaultMessage: 'Dive Deeper with Elasticsearch',
+          })}
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="l" />
+      <EuiFlexGroup direction={currentBreakpoint === 'xl' ? 'row' : 'column'}>
+        <EuiFlexItem>
+          <DocCallouts
+            title={i18n.translate('xpack.searchHomepage.searchLabs.title', {
+              defaultMessage: 'Search Labs',
+            })}
+            description={i18n.translate('xpack.searchHomepage.searchLabs.description', {
+              defaultMessage:
+                'Explore the latest articles and tutorials on using Elasticsearch for AI/ML-powered search experiences.',
```

Comment:
The search-labs site says " Start with the Elasticsearch docs to learn the fundamentals. When you're ready, come back to Elasticsearch Labs for more advanced content using our latest innovations..." so this is an attempt to convey the same here:
```suggestion
                'Explore the latest innovations for advanced search experiences.',
```